modules/efs/main.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  eks_cluster_managed_security_group_id = module.eks.outputs.eks_cluster_managed_security_group_id
+  existing_security_groups = var.use_eks_security_group ? [module.eks[0].outputs.eks_cluster_managed_security_group_id] : []
```

Comment:
Is the count logic set on the eks remote state? Otherwise the 0 index would throw an error.

Reply:
Ah, yes. That's what I was seeing with https://github.com/cloudposse/terraform-aws-components/pull/346#discussion_r684980771

Thanks. Works now.